security/pkg/credential/token.go

```diff
@@ -42,7 +42,7 @@ func (fetcher *GcpTokenFetcher) FetchToken() (string, error) {
 	return metadata.Get(fetcher.getTokenURI())
 }
 
-// FetchServiceAccount fetches the GCE VM identity jwt token from its metadata server.
+// FetchServiceAccount fetches the GCE VM service account.
```

Comment:
why updating the doc? I think the previous is more accurate. you can extend it by JWT of the VM service account.

Reply:
Currently this function returns the SA of the VM instance, right? Then it strings together with `spiffe.GenSpiffeURI` (calls `FetchServiceAccount`) to construct a spiffe identity.